README.md

```diff
@@ -1,7 +1,6 @@
 # Certificate Generation and Validation Using Blockchain
 
-[![Docker Build](https://img.shields.io/docker/automated/thakursaurabh1998/blockchain-certification.svg?style=for-the-badge)](https://hub.docker.com/r/thakursaurabh1998/blockchain-certification)
-[![Docker Build Status](https://img.shields.io/docker/build/thakursaurabh1998/blockchain-certification.svg?style=for-the-badge)](https://hub.docker.com/r/thakursaurabh1998/blockchain-certification)
+[![Docker Build](https://img.shields.io/docker/automated/thakursaurabh1998/blockchain-certification.svg?style=for-the-badge)](https://hub.docker.com/r/yogdaan/blockchain-certification)
```

Comment:
replace the badge with https://img.shields.io/docker/pulls/yogdaan/blockchain-certification?label=Docker%20Pulls&style=flat-square